html/comparison.html.haml

```diff
@@ -178,7 +178,7 @@
                 %span.summary-part{"ng-show" => "diff.patterns.only_in2"}
                   {{diff.meta.description_b}}: {{diff.patterns.only_in2.length}} patterns
                 %span.summary-part{"ng-show" => "diff.patterns.common"}
-                  %a{:class=>"show-common-elements", href: "#patterns_both"}
+                  %a{:class=>"show-common-elements", href: "#patterns_both"}<
```

Comment:
Avoid defining `class` in attributes hash for static class names<br>Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace